crates/cxx-qt-gen/src/generator/cpp/mod.rs

```diff
@@ -92,20 +91,17 @@ pub fn get_cpp_params(method: &ForeignItemFn, type_names: &TypeNames) -> Result<
             if let FnArg::Typed(PatType { pat, ty, .. }) = input {
                 let ident = if let Pat::Ident(PatIdent { ident, .. }) = &**pat {
                     ident
-                } else {
-                    return Err(Error::new(input.span(), "Unknown pattern for type"));
+                }
+                // CODECOV_EXCLUDE_START
+                else {
+                    unreachable!("Unknown pattern for type, FnArg can only have Pat::Ident")
                 };
+                // CODECOV_EXCLUDE_STOP
```

Comment:
in other places you have done

```
                    } else {
                        // CODECOV_EXCLUDE_START
                        unreachable!(...);
                        // CODECOV_EXCLUDE_STOP
                    }
```

which then doesn't break the coding style of having `} else {` instead of `else` on it's own line. Can the same be done here?

Reply:
yep, fixed